lib/fog/compute/google/models/instance_templates.rb

```diff
@@ -0,0 +1,22 @@
+module Fog
+  module Compute
+    class Google
+      class InstanceTemplates < Fog::Collection
+        model Fog::Compute::Google::InstanceTemplate
+
+        def all
+          data = service.list_instance_templates.items
+          load(data.map(&:to_h))
+        end
+
+        def get(identity)
+          if instance_template = service.get_instance_template(identity)
+            new(instance_template.to_h)
+          end
+        rescue Fog::Errors::NotFound
```

Comment:
You should rescue `Google::Apis::ClientError: notFound`, this currently doesn't return nil as it should.

For an example see:
https://github.com/fog/fog-google/blob/985cd021bcfe4f403deb7c0077997529d529b4b1/lib/fog/compute/google/models/servers.rb#L29-L42

Reply:
Done